build-system/server/new-server/transforms/transform.ts

```diff
@@ -22,14 +22,14 @@ import transformScriptPaths from './scripts/scripts-transform';
 import transformStories from './stories/stories-transform';
 import transformCss from './css/css-transform';
 
-const transforms = [transformStories, transformScriptPaths, transformCss];
-
 const argv = minimist(process.argv.slice(2));
-if (argv.esm) {
-  transforms.unshift(transformModules);
-}
 
 export async function transform(fileLocation: string): Promise<string> {
+  const transforms = [transformStories, transformScriptPaths, transformCss];
```

Comment:
could we move this back out to the module level instead of the function level?

Reply:
Done.